docs/source/supergraphs.md

```diff
@@ -61,6 +61,14 @@ subgraphs:
       subgraph: actors
 ```
 
+#### Using Federation 2
```

Comment:
```suggestion
#### Using with Federation 2
```

I suggest also adding `federation_version: 2` to the YAML example just above here